### DISCUSSION TITLE
Welcome to the posit::conf(2024) "Intro to MLOps with vetiver" workshop 🏺

### DISCUSSION THREAD

**Discussion by isabelizimm** [posit-conf-2024/vetiver#12] [Announcements]
Welcome! 🎉 

We’re using GitHub Discussions as a place to communicate during [our posit::conf(2024) workshop](https://posit-conf-2024.github.io/vetiver). We hope that you use this space to ask questions, share ideas, and engage with your fellow workshop participants. Be sure to review the [Code of Conduct](https://posit.co/code-of-conduct/) for this conference and our workshop.

To get started, please introduce yourself, your role, where you're coming from, and something you're excited about for posit::conf(2024)!

**Comment by kyleGrealis** [comment]
Hi, I'm Kyle Grealis and I'm a biostatician & data analyst at the University of Miami. My team has been conducting research on opioid addiction & treatment, so having access to deployed models will increase our ability to collaborate while working from many different locations.
 Since I'm pretty new to Python but have experience with R, it will be exciting to sharpen some new skills while learning from the greats 😊 See you all soon!!

**Comment by AdamSampson-BF** [comment]
Hi all. I'm Adam Sampson and I'm a Lead Data Scientist from Brown-Forman Corporation in Louisville, Kentucky. I work on a variety of modeling projects for our sales and planning teams to grow our premium alcohol brands using data driven decisions. We use R frequently and I'm looking forward to learning more about improving our processes through MLOps to ensure our models continue to drive the best possible decisions over time. I look forward to meeting you all.
